docs/schema.md

```diff
@@ -636,6 +647,7 @@ The defined ecosystems are:
 | `Bioconductor` | The biological R package ecosystem. The `name` is an R package name. |
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `ConanCenter` | The ConanCenter ecosystem for C and C++; the `name` field is a Conan package name.  |
+| `CGA` | The Chainguard package ecosystem; the `name` is the name of the source package. The package name might have a `:<RELEASE>` suffix to scope the package to a particular version stream. `<RELEASE>` is a numeric version. |
```

Comment:
My two cents here — 

1. We don't have source packages at this time, so there's not a valuable semantic in distinguishing these here IMHO.
2. We don't have Debian-style version streams. We do have version streams that add suffixes to the package name, but it's not meaningful for the advisory data or for vulnerability matching at scan time — the package name should be used verbatim (even if a version stream suffix is present).

```suggestion
| `CGA` | The Chainguard package ecosystem; the `name` is the name of the package. |
```

Reply:
i've add that to the commit, i did similar text before, so i think we are aligned. i was just not sure